airbyte-integrations/connectors/destination-snowflake/build.gradle

```diff
@@ -27,7 +27,7 @@ application {
 dependencies {
     implementation 'com.google.cloud:google-cloud-storage:1.113.16'
     implementation 'com.google.auth:google-auth-library-oauth2-http:0.25.5'
-    implementation 'net.snowflake:snowflake-jdbc:3.13.9'
+    implementation 'net.snowflake:snowflake-jdbc:3.13.19'
```

Comment:
Can you add a comment here so that next time someone tries to upgrade the driver, they can easily know about the problem. Something like what you mentioned in the PR description:

`Updating to any newer versions is causing a regression with normalization` + a summary of the regression.

Reply:
> Can you add a comment here so that next time someone tries to upgrade the driver, they can easily know about the problem. Something like what you mentioned in the PR description:
> 
> `Updating to any newer versions is causing a regression with normalization` + a summary of the regression.

Thanks, I've completed the relevant pre-merge items. Please take a look